artifacts/ArtifactHandling/Invoke-DownloadArtifact.ps1

```diff
@@ -121,28 +121,36 @@ function Invoke-DownloadArtifact {
             try {
                 $started = Get-Date -Format "o"
                 if ($downlodUrl.StartsWith("http")) {
+                    $actionMessage = "downloaded"
                     try {
                         Invoke-WebRequest -Method Get -uri $downlodUrl -OutFile "$archive" -Headers $headers
                     } catch {
                         Invoke-WebRequest -Method Get -uri $downlodUrl -OutFile "$archive"
                     }
                 } else {
-                    Copy-Item -Path $downlodUrl -Destination "$archive"
+                    $actionMessage = "copied"
+                    if (! (Test-Path -Path $downlodUrl)) {
+                        Add-ArtifactsLog -message "Artifact '$downlodUrl' does not exist" -severity Warn -success skip
+                    } else {
+                        Copy-Item -Path $downlodUrl -Destination "$archive"
```

Comment:
@megel $archive is always a .zip, I think. Therefore I worked on something similar with this: https://github.com/cosmoconsult/container-scripts-for-selfservice/compare/stability-improvements